serverless/serverless
# Spotinst - updating templates and accompanying docs

## What did you implement:

Updated our templates for Node, python, and ruby. Updated the docs that describe these new features



## How did you implement it:

Mostly a cosmetic change 



## How can we verify it:

Check the spotinst templates for node python and ruby 
Check the User guide and examples in the Spotinst Docs



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
